### PR TITLE
finished project and passed all tests

### PR DESCRIPTION
**Pull request by ahetrick22**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1,66 @@
...
+  var books = [];                       //private array of book objects
+
+//adds a book object to the private array (created with the Book module)
+  var addBook = function(bookObject) {
+    books.push(bookObject);
+  };
+
+//only checks out if it exists and not already checked out
+  var checkOutBook = function(book) {
+    if (books.indexOf(book) > -1 && book.getAttribute("checkedOut") === false) {
```
Comment (trblackw) [inline main.js:12]: This is a different approach to the conditional--I like it! Though I think using a for loop would be a bit simpler and easier to read.

File: main.js [hunk 2]
```diff
@@ -1 +1,66 @@
...
+      book.setAttribute("checkedOut", true);
+      return;
+    }
+    console.log("That book doesn't exist in your library or is already checked out");
+  };
+
+//only returns if exists and already checked out
+  var returnBook = function(book) {
```
Comment (trblackw) [inline main.js:20]: I would stick to your previous style for consistency sake...
i.e.
```
if (books.indexOf(book) > -1 && book.getAttribute("checkedOut") === false) {
          book.setAttribute("checkedOut", true);
      return;
    }
    console.log("That book doesn't exist in your library or is already checked out");
  };
```

File: main.js [hunk 3]
```diff
@@ -1 +1,66 @@
...
+    if (books.indexOf(book) > -1 && book.getAttribute("checkedOut") === true) {
+      book.setAttribute("checkedOut", false);
+    } else {
+      console.log("That book doesn't exist in your library or is already returned");
+    }
+  };
+
+//library returns with 3 total methods that allow for closure when invoked
```
Comment (trblackw) [inline main.js:28]: good comment

File: main.js (hunk 1 above)
Comment (ahetrick22) [reply main.js:12]: Thanks for these comments!

**Pull request closed without merging**